src/applications/vaos/services/appointment/index.js

```diff
@@ -344,33 +326,23 @@ export function isUpcomingAppointmentOrExpressCare(appt) {
     );
   }
 
-  return (
-    appt.vaos.isExpressCare &&
-    appt.status !== APPOINTMENT_STATUS.fulfilled &&
-    moment(appt.start).isAfter(
-      // going one day back to account for late in the day EC requests
-      moment()
-        .startOf('day')
-        .add(-1, 'day'),
-    )
-  );
+  return appt.status !== APPOINTMENT_STATUS.fulfilled;
 }
 
 /**
  * Returns true if the given Appointment is a canceled confirmed appointment
- * or a canceled Express Care request
  *
  * @export
  * @param {Appointment} appt The FHIR Appointment to check
  * @returns {boolean} Whether or not the appointment is a canceled
- *  appointment or Express Care request
+ *  appointment
  */
-export function isCanceledConfirmedOrExpressCare(appt) {
+export function isCanceledConfirmed(appt) {
   const today = moment();
 
   if (
-    CONFIRMED_APPOINTMENT_TYPES.has(appt.vaos.appointmentType) ||
-    appt.vaos.isExpressCare
+    CONFIRMED_APPOINTMENT_TYPES.has(appt.vaos.appointmentType) &&
+    !appt.vaos.isExpressCare
```

Comment:
You can remove this `!appt.vaos.isExpressCare` check, the confirmed appointments check will filter out EC requests on its own.